gogs/files/templates/gogs-persistent-template.yaml

```diff
@@ -0,0 +1,385 @@
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    description: The Gogs git server (https://gogs.io/)
+    tags: instant-app,gogs,go,golang
+  name: gogs
+objects:
+- kind: SecurityContextConstraints
+  apiVersion: v1
+  metadata:
+    annotations:
+      kubernetes.io/description: anyuid provides all features of the restricted SCC
+        but allows users to run with any UID and any GID.
+    name: anyuid
+  users:
+  - system:serviceaccount:gogs:${APPLICATION_NAME}
```

Comment:
Can we ensure we don't "hardcode" the namespace to `gogs`? I.e.: use a parameter for the namespace and default it to `gogs`

Reply:
We need to remove projects.yml then and parametrize the 'namespace' for the application. Is this what you are looking for?

Reply:
Yes please. We’d most likely utilize templates like this in other inventories, in combination with other tool, and in those cases most likely using a different namespace.

Reply:
(just like what we did with the GitLab template)